src/vue/fields/SelectField.vue

```diff
@@ -84,38 +96,34 @@ export default {
       type: Boolean,
       default: false,
     },
-    formFree: {
-      type: Boolean,
-      required: false,
-      default: false,
-    },
     errorMessage: {
       type: String,
-      default: undefined,
+      default: '',
+    },
+    keyAsValueText: {
```

Comment:
i can't figure out what does it serves for

Reply:
described in the doc